purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -80,23 +80,25 @@ open class PurchasesConfiguration(builder: Builder) {
         }
 
         /**
-         * **Feature in beta**. Enables signature verification of requests to the RevenueCat backend
-         * in informational mode and enables diagnostics reports to RevenueCat to help us analyze this feature.
-         * Informational mode means that the behavior will remain the same but we will provide information about
-         * detected attempts of hacking.
+         * Deprecated. Use [entitlementVerificationMode] instead.
+         *
+         * Enables signature verification of requests to the RevenueCat backend
+         * and enables diagnostics reports to RevenueCat to help us analyze this feature.
          *
          * When changing from disabled to enabled, the SDK will clear the CustomerInfo cache.
          * This means users will need to connect to the internet to get their entitlements back.
          *
          * The result of the verification can be obtained from [EntitlementInfos.verification] or
          * [EntitlementInfo.verification].
          *
-         * This feature is currently in beta and the behavior may change. Only available in Kotlin.
-         *
          * Default mode is disabled.
          *
          * @warning This function is marked as [ExperimentalPreviewRevenueCatPurchasesAPI] and may change in the future.
          */
+        @Deprecated(
+            "Use the new entitlementVerificationMode setter instead.",
+            ReplaceWith("entitlementVerificationMode(EntitlementVerificationMode.INFORMATIONAL)"),
+        )
         @JvmSynthetic
         @ExperimentalPreviewRevenueCatPurchasesAPI
         fun informationalVerificationModeAndDiagnosticsEnabled(enabled: Boolean) = apply {
```

Comment:
I kept the existing experimental method but deprecated, In case anyone is using it since it's pretty easy to keep. We can remove it on the next major.